src/pal/src/loader/module.cpp

```diff
@@ -728,18 +728,19 @@ PAL_UnregisterModule(
 
 Parameters:
     IN hFile - file to map
+    IN prevLoadedBase - previously loaded base
```

Comment:
Is it possible to set this argument to non-`NULL` in a scenario other than "preload ...  finish load"?
If not, it is better to add a comment that would indicate this.

It would be even better to have a comment with more detailed description of the preloading scheme, and to reference the comment from here and other places like this.